hibernate-core/src/main/java/org/hibernate/dialect/Oracle8iDialect.java

```diff
@@ -490,7 +490,7 @@ public boolean forUpdateOfColumns() {
 
 	@Override
 	public String getQuerySequencesString() {
-		return "select * from all_sequences";
+		return "select * from user_sequences";
```

Comment:
In Oracle, 'all_sequences' is a view to list everything regardless of schema, whereas 'user_sequences' will limit within current schema. User and schema is the same thing in Oracle.